engine/runtime/src/test/java/org/enso/interpreter/test/TestBase.java

```diff
@@ -32,6 +32,14 @@ protected static Context createDefaultContext() {
     return context;
   }
 
+  protected static Context createNonStrictContext(OutputStream out) {
```

Comment:
Is there only single usage of this method? Consider using `createDefaultContext` and apply the `STRICT_ERRORS` option to it.